src/nvme/types.h

```diff
@@ -1253,6 +1259,7 @@ enum nvme_id_ctrl_oacs {
 	NVME_CTRL_OACS_VIRT_MGMT		= 1 << 7,
 	NVME_CTRL_OACS_DBBUF_CFG		= 1 << 8,
 	NVME_CTRL_OACS_LBA_STATUS		= 1 << 9,
+	NVME_CTRL_OACS_CMD_FEAT_LD		= 1 << 10,
```

Comment:
If I am not completely mistaken, this has nothing to do with DS, right? Could you split these changes into smaller commits which describe each changes.